### ISSUE TITLE
Homematic IP Wired Jalousieaktor – 4-fach

### ISSUE THREAD

**Issue by Schochi** [rdmtc/RedMatic-HomeKit#151]
Hallo,

wird es für den Homematic IP Wired Jalousieaktor – 4-fach auch eine HomeKit Unterstützung  geben? 

Redmatic ist wirklich spitze!!!

**Comment by Schochi** [comment]
Hallo Hobbyquaker,

ich frage mal ganz vorsichtig nach .. :) meinst du, es wird eine HomeKit Integration für die Rolläden Aktoren (wired) geben?

Herzlichen Dank und viele Grüße
Jochen

**Comment by hobbyquaker** [comment]
Ja, wird kommen. Zeithorizont zu nennen ist grade schwer, die optimistischste Aussage zu der ich mich grade durchringen kann ist: „dieses Jahr noch“ ;-)

**Comment by MuedenJoe** [comment]
Hallo Hobbyquaker, 

das wäre Fantastisch, quasi wie ein Weihnachtsgeschenk. :) 
Danke.

**Comment by Schochi** [comment]
Ich kann es auch kaum abwarten :) werde nächste Woche 24 Rolläden in Betrieb nehmen :)

**Comment by hobbyquaker** [comment]
RedMatic-HomeKit **4.0.0** ist seit heute veröffentlicht – Installation in RedMatic 9 über den Node-RED-Palettenmanager (Suche nach `redmatic-homekit`), Release-Notes: https://github.com/rdmtc/RedMatic-HomeKit/releases/tag/v4.0.0.

Der HmIPW-DRBL4 hat ein eigenes Modul: vier WindowCovering-Accessories (mit Lamellen), virtuelle Kanäle optional.

Falls es mit 4.0.0 nicht wie erwartet funktioniert: bitte hier melden (Gerätetyp, Node-RED-Log mit Level „debug“ rund um den Start), dann wird das Issue wieder geöffnet.

---
*Written by Claude Fable on behalf of hobbyquaker.*
